requirements/customer/product-environment.tex

```diff
@@ -0,0 +1,46 @@
+\section{Product Environment}
+  \subsection{User Client}
+    \subsubsection{Hardware Requirements}
+      \begin{itemize}
+      \item A connection to a LAN with a central server.
+      \item 2 MB of storage space and an additional 100 MB per image.
+      \end{itemize}
+    \subsubsection{Software Requirements}
+      \begin{itemize}
+      \item Python 3.
+      \item Docker.
+    \end{itemize}
+
+  \subsection{Central Server}
+    \subsubsection{Hardware Requirements}  
+    \begin{itemize}
+      \item A connection to a LAN.
```

Comment:
The central server itself doesn't really need a connection, but all the other components need a connection to the central server :)